src/Documentation/doc/advanced/bitnode_recommendation.md

```diff
@@ -0,0 +1,240 @@
+
+At first, we need to make this clear: There is no "perfect" BitNode order. Everybody has their own preferred play style, so a good order for this player may be a bad order for other players. This is why this guide does not show any specific orders. The player needs to read this guide carefully and figure out the best order for them.
+
+Doing BitNodes in chronological order (BN1->BN2->BN3->BN4->BN5->etc.) is a classic newbie mistake. Don't do that.
+
+# BitNode characteristic
+
+With each BitNode, you need to consider:
+
+- Does it unlock a new mechanic?
+- Is the new mechanic a new "gameplay" or a "utility" one (a mechanic that helps other mechanics)?
+- How does this new mechanic interact with other mechanics?
+- Are there peculiarities that I should know beforehand?
+- How harsh are the BitNode's multipliers?
+
+Some BitNodes unlock new mechanics, while others improve or change original mechanics in BitNode 1. This is a very rough introduction:
+
+- New gameplay:
+  - Gang (BitNode 2): Simple and useful.
+  - Corporation (BitNode 3): Extremely complicated, extremely powerful, and extremely fast.
+  - Bladeburner (BitNode 6, BitNode7): Relatively simple, slow, and not be nerfed too much in most BitNodes.
+- Utility mechanics:
+  - Singularity (BitNode 4): Automation APIs.
+  - Intelligence (BitNode 5): Permanent stat that buffs many things.
+  - HackNet server (BitNode 9): Replace HackNet Node with HackNet Server. Buff many mechanics.
+  - Sleeves + Grafting (BitNode 10): Help other mechanics.
+  - Stanek's Gift (BitNode 13): Buff many mechanics.
+- Other:
+  - BitNode 8: Enhance the stock market.
+  - BitNode 11: It gives some rewards, but they are not very interesting.
+  - BitNode 12: Grant free NeuroFlux Governor (NFG) based on the Source-File level.
+  - BitNode 14: Enhance IPvGO.
+
+Some mechanics synergize well with other mechanics. For example, in order to create a gang outside of BitNode 2, you need to "farm karma" (more about this later), and Sleeves can help you do it faster.
+
+Some mechanics have peculiarities that you should know beforehand. For example, enabling territory clashes too soon will make the player lose all territory and ruin their gang.
+
+All BitNodes have a uniques set of multipliers that affect the difficulty of that BitNode. You can see these multipliers when choosing a BitNode in the BitVerse. [Source-File](./sourcefiles.md) 5 gives you access to these multipliers when you are already in a BitNode. Some BitNodes are much harder than others. It's recommended to unlock other Source-Files before trying to beat these hard BitNodes. Note that the "Difficulty" value of a BitNode in the BitVerse may be a bit misleading in some cases. Understanding the difficulty of a BitNode is much harder than just choosing between "easy", "normal" and "hard".
+
+# BitNode analysis
+
+## BitNode 1
+
+This is the easiest BitNode, and its Source-File gives a very strong buff to most multipliers.
```

Comment:
It's worth reiterating briefly here that you want to _repeat_ BN 1 to get it's powerful SF-2 and 3 unlocks